.ci/pull-request-config.yaml

```
@@ -0,0 +1,12 @@
+version: "2.0"
+
+dependencies: ./project-dependencies.yaml
+
+build:
+  - project: kiegroup/optaplanner
+    build-command:
+      upstream: mvn -e clean install -DskipTests -Denforcer.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3
+
+  - project: kiegroup/optaplanner-quickstarts
+    build-command:
+      current: mvn -e -nsu -Dfull -Pwildfly clean install -Prun-code-coverage  -Dcontainer.profile=wildfly -Dcontainer=wildfly -Dintegration-tests=true -Dmaven.test.failure.ignore=true -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3
```

Removing irrelevant profiles and properties.
```suggestion
      current: mvn -e -nsu -Dfull clean install -Dmaven.test.failure.ignore=true -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3
```